src/test/resources/features/15-attachment/attachments.feature

```diff
@@ -0,0 +1,29 @@
+ Feature: Attachments Management
```

Comment:
Please test this locally, i have tried doing this and all the 3 scenarios were failing

```
Tests in error: 
  Uploading Attachment(Attachments Management)
  Removing un intended Attachment(Attachments Management)
  Capturing an image(Attachments Management)

```

Reply:
Hey @kaweesi is there a way @CucumberOptions can allow running selenium test against a single file and ignore other files present in the same folder. After making a research on this, glue option can be used to define the definition step package.  
Also i would like to test the feature under construction on my local instance of standalone. Is this possible i can see in the test.properties file the web-url redirects at the remote server. If so is it ok to do some changes in this file to meet my requirement. Thanks